content/en/logs/log_collection/csharp.md

```diff
@@ -423,15 +577,23 @@ In the `Serilog.WriteTo` array, add an entry for `DatadogLogs`. An example is sh
 }
 ```
 
-[1]: https://www.nuget.org/packages/Serilog.Sinks.Datadog.Logs
-[2]: https://app.datadoghq.com/organization-settings/api-keys
-[3]: /logs/log_configuration/attributes_naming_convention/#reserved-attributes
-{{% /tab %}}
-{{< /tabs >}}
-
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: /logs/log_configuration/parsing
-[2]: /tracing/connect_logs_and_traces/dotnet/
+[1]: #file-tail-logging-with-the-datadog-agent
+[2]: #agentless-logging-with-apm
+[3]: #agentless-logging-with-serilog-sink
+[4]: /logs/log_configuration/parsing
+[5]: /tracing/connect_logs_and_traces/dotnet/
+[6]: /agent/logs/advanced_log_collection
+[7]: /serverless/azure_app_services
+[8]: /account_management/org_settings/sensitive_data_detection/#overview
+[9]: /tracing/setup_overview/setup/dotnet-core
+[10]: /tracing/setup_overview/setup/dotnet-framework
+[11]: /organization-settings/api-keys
```

Comment:
```suggestion
[11]: https://app.datadoghq.com/organization-settings/api-keys
```

Reply:
Currently the API keys link is giving a 404 error. I believe this link is meant to open the API keys page within the user's Datadog account -- is this correct? If so, relative links only work for pages in the documentation repo. 
If the intention is instead to open documentation page for API keys, that path is /account_management/api-app-keys/#api-keys
Apologies for not catching this sooner, could you confirm and update as needed?

Reply:
Oops, yes, that's my fault. I tried to "fix" it in the previous changes 🤦‍♂️, your revert is correct, thanks!